2019/05/15 11:45:13 AM:INFO:Qt version:5.11.1
2019/05/15 11:45:13 AM:INFO:PyQt version:5.11.2
2019/05/15 11:45:13 AM:INFO:SIP version:4.19.7
2019/05/15 11:45:14 :DEBUG:13418: gpg --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --version
2019/05/15 11:45:14 :DEBUG:stderr reader: <Thread(Thread-1, initial daemon)>
2019/05/15 11:45:14 :DEBUG:stdout reader: <Thread(Thread-2, initial daemon)>
2019/05/15 11:45:14 :DEBUG:chunk: b'gpg (GnuPG) 2.2.4\nlibgcrypt 1.8.1\nCopyright (C) 2017 Free Software Foundation, Inc.\nLicense GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>\nThis is free software: you are free to change and redistribute it.\nThere is NO WARRANTY, to t'
2019/05/15 11:45:14 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:14 :INFO:Connection to database established
2019/05/15 11:45:14 :INFO:Created table passTable: 
2019/05/15 11:45:14 :INFO:Created table configs: 
2019/05/15 11:45:14 :INFO:Created table email: 
2019/05/15 11:45:14 :INFO:Created table keys: 
2019/05/15 11:45:14 :INFO:All tables were created/exist!
2019/05/15 11:45:14 :INFO:Getting row: 1 from table: configs
2019/05/15 11:45:14 :INFO:Getting row: 1 from table: configs
2019/05/15 11:45:14 :DEBUG:13421: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:14 :DEBUG:Wrote passphrase
2019/05/15 11:45:14 :DEBUG:data copier: <Thread(Thread-3, initial daemon)>, <_io.BytesIO object at 0x7f292cfc6048>, <_io.BufferedWriter name=27>
2019/05/15 11:45:14 :DEBUG:stderr reader: <Thread(Thread-4, initial daemon)>
2019/05/15 11:45:14 :DEBUG:closed output, 179 bytes sent
2019/05/15 11:45:14 :DEBUG:stdout reader: <Thread(Thread-5, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: AES256 encrypted data
2019/05/15 11:45:15 :DEBUG:chunk: b'Y8EektQJriiQL2Z9'
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NEED_PASSPHRASE_SYM 9 3 2
2019/05/15 11:45:15 :DEBUG:gpg: encrypted with 1 passphrase
2019/05/15 11:45:15 :DEBUG:[GNUPG:] BEGIN_DECRYPTION
2019/05/15 11:45:15 :DEBUG:[GNUPG:] DECRYPTION_COMPLIANCE_MODE 23
2019/05/15 11:45:15 :DEBUG:message ignored: DECRYPTION_COMPLIANCE_MODE, 23
2019/05/15 11:45:15 :DEBUG:[GNUPG:] DECRYPTION_INFO 2 9
2019/05/15 11:45:15 :DEBUG:[GNUPG:] PLAINTEXT 62 1557913385
2019/05/15 11:45:15 :DEBUG:[GNUPG:] PLAINTEXT_LENGTH 16
2019/05/15 11:45:15 :DEBUG:[GNUPG:] DECRYPTION_OKAY
2019/05/15 11:45:15 :DEBUG:[GNUPG:] GOODMDC
2019/05/15 11:45:15 :DEBUG:[GNUPG:] END_DECRYPTION
2019/05/15 11:45:15 :DEBUG:decrypt result: b'Y8EektQJriiQL2Z9'
2019/05/15 11:45:15 :INFO:Getting everything from passTable
2019/05/15 11:45:15 :DEBUG:13425: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:15 :DEBUG:Wrote passphrase
2019/05/15 11:45:15 :DEBUG:data copier: <Thread(Thread-6, initial daemon)>, <_io.BytesIO object at 0x7f292cfcb0a0>, <_io.BufferedWriter name=27>
2019/05/15 11:45:15 :DEBUG:stderr reader: <Thread(Thread-7, initial daemon)>
2019/05/15 11:45:15 :DEBUG:closed output, 1 bytes sent
2019/05/15 11:45:15 :DEBUG:stdout reader: <Thread(Thread-8, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: [don't know]: invalid packet (ctb=31)
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 3
2019/05/15 11:45:15 :DEBUG:decrypt result: b''
2019/05/15 11:45:15 :DEBUG:13429: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:15 :DEBUG:Wrote passphrase
2019/05/15 11:45:15 :DEBUG:data copier: <Thread(Thread-9, initial daemon)>, <_io.BytesIO object at 0x7f292cfcb0a0>, <_io.BufferedWriter name=27>
2019/05/15 11:45:15 :DEBUG:stderr reader: <Thread(Thread-10, initial daemon)>
2019/05/15 11:45:15 :DEBUG:closed output, 163 bytes sent
2019/05/15 11:45:15 :DEBUG:stdout reader: <Thread(Thread-11, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: AES256 encrypted data
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NEED_PASSPHRASE_SYM 9 3 2
2019/05/15 11:45:15 :DEBUG:gpg: encrypted with 1 passphrase
2019/05/15 11:45:15 :DEBUG:chunk: b'test'
2019/05/15 11:45:15 :DEBUG:[GNUPG:] BEGIN_DECRYPTION
2019/05/15 11:45:15 :DEBUG:[GNUPG:] DECRYPTION_COMPLIANCE_MODE 23
2019/05/15 11:45:15 :DEBUG:message ignored: DECRYPTION_COMPLIANCE_MODE, 23
2019/05/15 11:45:15 :DEBUG:[GNUPG:] DECRYPTION_INFO 2 9
2019/05/15 11:45:15 :DEBUG:[GNUPG:] PLAINTEXT 62 1557913404
2019/05/15 11:45:15 :DEBUG:[GNUPG:] PLAINTEXT_LENGTH 4
2019/05/15 11:45:15 :DEBUG:[GNUPG:] DECRYPTION_OKAY
2019/05/15 11:45:15 :DEBUG:[GNUPG:] GOODMDC
2019/05/15 11:45:15 :DEBUG:[GNUPG:] END_DECRYPTION
2019/05/15 11:45:15 :DEBUG:decrypt result: b'test'
2019/05/15 11:45:15 :DEBUG:13435: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:15 :DEBUG:Wrote passphrase
2019/05/15 11:45:15 :DEBUG:data copier: <Thread(Thread-12, initial daemon)>, <_io.BytesIO object at 0x7f292cfcb0a0>, <_io.BufferedWriter name=27>
2019/05/15 11:45:15 :DEBUG:stderr reader: <Thread(Thread-13, initial daemon)>
2019/05/15 11:45:15 :DEBUG:closed output, 26 bytes sent
2019/05/15 11:45:15 :DEBUG:stdout reader: <Thread(Thread-14, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:15 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:15 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:15 :DEBUG:decrypt result: b''
2019/05/15 11:45:15 :DEBUG:13439: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:15 :DEBUG:Wrote passphrase
2019/05/15 11:45:15 :DEBUG:data copier: <Thread(Thread-15, initial daemon)>, <_io.BytesIO object at 0x7f292cfcb0a0>, <_io.BufferedWriter name=27>
2019/05/15 11:45:15 :DEBUG:stderr reader: <Thread(Thread-16, initial daemon)>
2019/05/15 11:45:15 :DEBUG:closed output, 5 bytes sent
2019/05/15 11:45:15 :DEBUG:stdout reader: <Thread(Thread-17, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:15 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:15 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:15 :DEBUG:decrypt result: b''
2019/05/15 11:45:15 :DEBUG:13443: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:15 :DEBUG:Wrote passphrase
2019/05/15 11:45:15 :DEBUG:data copier: <Thread(Thread-18, initial daemon)>, <_io.BytesIO object at 0x7f292cfcb0a0>, <_io.BufferedWriter name=27>
2019/05/15 11:45:15 :DEBUG:stderr reader: <Thread(Thread-19, initial daemon)>
2019/05/15 11:45:15 :DEBUG:closed output, 13 bytes sent
2019/05/15 11:45:15 :DEBUG:stdout reader: <Thread(Thread-20, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:15 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:15 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:15 :DEBUG:decrypt result: b''
2019/05/15 11:45:15 :DEBUG:13447: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:15 :DEBUG:Wrote passphrase
2019/05/15 11:45:15 :DEBUG:data copier: <Thread(Thread-21, initial daemon)>, <_io.BytesIO object at 0x7f292cfcb0a0>, <_io.BufferedWriter name=27>
2019/05/15 11:45:15 :DEBUG:stderr reader: <Thread(Thread-22, initial daemon)>
2019/05/15 11:45:15 :DEBUG:closed output, 4 bytes sent
2019/05/15 11:45:15 :DEBUG:stdout reader: <Thread(Thread-23, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:15 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:15 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:15 :DEBUG:decrypt result: b''
2019/05/15 11:45:15 :DEBUG:13451: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:15 :DEBUG:Wrote passphrase
2019/05/15 11:45:15 :DEBUG:data copier: <Thread(Thread-24, initial daemon)>, <_io.BytesIO object at 0x7f292cfcb0a0>, <_io.BufferedWriter name=27>
2019/05/15 11:45:15 :DEBUG:stderr reader: <Thread(Thread-25, initial daemon)>
2019/05/15 11:45:15 :DEBUG:closed output, 5 bytes sent
2019/05/15 11:45:15 :DEBUG:stdout reader: <Thread(Thread-26, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:15 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:15 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:15 :DEBUG:decrypt result: b''
2019/05/15 11:45:15 :DEBUG:13455: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:15 :DEBUG:Wrote passphrase
2019/05/15 11:45:15 :DEBUG:data copier: <Thread(Thread-27, initial daemon)>, <_io.BytesIO object at 0x7f292cfcb0a0>, <_io.BufferedWriter name=27>
2019/05/15 11:45:15 :DEBUG:stderr reader: <Thread(Thread-28, initial daemon)>
2019/05/15 11:45:15 :DEBUG:closed output, 4 bytes sent
2019/05/15 11:45:15 :DEBUG:stdout reader: <Thread(Thread-29, initial daemon)>
2019/05/15 11:45:15 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:15 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:15 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:15 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:15 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:15 :DEBUG:decrypt result: b''
2019/05/15 11:45:30 :INFO:Getting everything from email
2019/05/15 11:45:30 :DEBUG:13469: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --symmetric --armor
2019/05/15 11:45:30 :DEBUG:Wrote passphrase
2019/05/15 11:45:30 :DEBUG:data copier: <Thread(Thread-30, initial daemon)>, <_io.BytesIO object at 0x7f2926590200>, <_io.BufferedWriter name=27>
2019/05/15 11:45:30 :DEBUG:stderr reader: <Thread(Thread-31, initial daemon)>
2019/05/15 11:45:30 :DEBUG:closed output, 11 bytes sent
2019/05/15 11:45:30 :DEBUG:stdout reader: <Thread(Thread-32, initial daemon)>
2019/05/15 11:45:31 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:31 :DEBUG:chunk: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCdUAxfmuGAOn/0kABRIgIm9aqFxoBj56RQCp4jrE5RI9P2+i2Tf3q9RdJ\nlLUuIIM/maZq+lD6WMsE5gUl0D0zMw5ImtF0NW0x1Cu/\n=6Um6\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:31 :DEBUG:[GNUPG:] NEED_PASSPHRASE_SYM 9 3 2
2019/05/15 11:45:31 :DEBUG:[GNUPG:] BEGIN_ENCRYPTION 2 9
2019/05/15 11:45:31 :DEBUG:[GNUPG:] END_ENCRYPTION
2019/05/15 11:45:31 :DEBUG:encrypt result: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCdUAxfmuGAOn/0kABRIgIm9aqFxoBj56RQCp4jrE5RI9P2+i2Tf3q9RdJ\nlLUuIIM/maZq+lD6WMsE5gUl0D0zMw5ImtF0NW0x1Cu/\n=6Um6\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:31 :DEBUG:13474: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --symmetric --armor
2019/05/15 11:45:31 :DEBUG:Wrote passphrase
2019/05/15 11:45:31 :DEBUG:data copier: <Thread(Thread-33, initial daemon)>, <_io.BytesIO object at 0x7f2926590200>, <_io.BufferedWriter name=27>
2019/05/15 11:45:31 :DEBUG:stderr reader: <Thread(Thread-34, initial daemon)>
2019/05/15 11:45:31 :DEBUG:closed output, 11 bytes sent
2019/05/15 11:45:31 :DEBUG:stdout reader: <Thread(Thread-35, initial daemon)>
2019/05/15 11:45:31 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:31 :DEBUG:chunk: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCXY5XI2VhhiX/0kABlY1qobjPoaaSTBnRPakuZfO9I86taQFk+iKqtASD\n91dH4LAzggXOyxy5j4/pExF5EgBiXROGEUureh0ZXsmv\n=ppHQ\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:31 :DEBUG:[GNUPG:] NEED_PASSPHRASE_SYM 9 3 2
2019/05/15 11:45:31 :DEBUG:[GNUPG:] BEGIN_ENCRYPTION 2 9
2019/05/15 11:45:31 :DEBUG:[GNUPG:] END_ENCRYPTION
2019/05/15 11:45:31 :DEBUG:encrypt result: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCXY5XI2VhhiX/0kABlY1qobjPoaaSTBnRPakuZfO9I86taQFk+iKqtASD\n91dH4LAzggXOyxy5j4/pExF5EgBiXROGEUureh0ZXsmv\n=ppHQ\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:31 :INFO:Inserted data into email table
2019/05/15 11:45:31 :DEBUG:13479: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --symmetric --armor
2019/05/15 11:45:31 :DEBUG:Wrote passphrase
2019/05/15 11:45:31 :DEBUG:data copier: <Thread(Thread-36, initial daemon)>, <_io.BytesIO object at 0x7f2926590200>, <_io.BufferedWriter name=27>
2019/05/15 11:45:31 :DEBUG:stderr reader: <Thread(Thread-37, initial daemon)>
2019/05/15 11:45:31 :DEBUG:closed output, 5 bytes sent
2019/05/15 11:45:31 :DEBUG:stdout reader: <Thread(Thread-38, initial daemon)>
2019/05/15 11:45:31 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:31 :DEBUG:chunk: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCQzAjIOFN+Nv/0joBodtdioCUQ7X8ghvDRzbosIkTKGnIIsIfZeSTtvQG\n3z54nuEg2wNbGIH/eUjntgrQ8E1Xz5sqTw8d\n=3naS\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:31 :DEBUG:[GNUPG:] NEED_PASSPHRASE_SYM 9 3 2
2019/05/15 11:45:31 :DEBUG:[GNUPG:] BEGIN_ENCRYPTION 2 9
2019/05/15 11:45:31 :DEBUG:[GNUPG:] END_ENCRYPTION
2019/05/15 11:45:31 :DEBUG:encrypt result: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCQzAjIOFN+Nv/0joBodtdioCUQ7X8ghvDRzbosIkTKGnIIsIfZeSTtvQG\n3z54nuEg2wNbGIH/eUjntgrQ8E1Xz5sqTw8d\n=3naS\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:31 :DEBUG:13484: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --symmetric --armor
2019/05/15 11:45:31 :DEBUG:Wrote passphrase
2019/05/15 11:45:31 :DEBUG:data copier: <Thread(Thread-39, initial daemon)>, <_io.BytesIO object at 0x7f2926590200>, <_io.BufferedWriter name=27>
2019/05/15 11:45:31 :DEBUG:stderr reader: <Thread(Thread-40, initial daemon)>
2019/05/15 11:45:31 :DEBUG:closed output, 6 bytes sent
2019/05/15 11:45:31 :DEBUG:stdout reader: <Thread(Thread-41, initial daemon)>
2019/05/15 11:45:32 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:32 :DEBUG:chunk: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCJ4Zsey+A+3v/0jgBAfdVw6lA4gVs1j2qRlRk1RNwH2Qnxe6I91FmzBZM\nu+B8sHB3QwRFwqhRUFwn7vNrF/qFYnEHpw==\n=lLhU\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:32 :DEBUG:[GNUPG:] NEED_PASSPHRASE_SYM 9 3 2
2019/05/15 11:45:32 :DEBUG:[GNUPG:] BEGIN_ENCRYPTION 2 9
2019/05/15 11:45:32 :DEBUG:[GNUPG:] END_ENCRYPTION
2019/05/15 11:45:32 :DEBUG:encrypt result: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCJ4Zsey+A+3v/0jgBAfdVw6lA4gVs1j2qRlRk1RNwH2Qnxe6I91FmzBZM\nu+B8sHB3QwRFwqhRUFwn7vNrF/qFYnEHpw==\n=lLhU\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:32 :DEBUG:13489: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --symmetric --armor
2019/05/15 11:45:32 :DEBUG:Wrote passphrase
2019/05/15 11:45:32 :DEBUG:data copier: <Thread(Thread-42, initial daemon)>, <_io.BytesIO object at 0x7f2926590200>, <_io.BufferedWriter name=27>
2019/05/15 11:45:32 :DEBUG:stderr reader: <Thread(Thread-43, initial daemon)>
2019/05/15 11:45:32 :DEBUG:closed output, 3 bytes sent
2019/05/15 11:45:32 :DEBUG:stdout reader: <Thread(Thread-44, initial daemon)>
2019/05/15 11:45:32 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:32 :DEBUG:chunk: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCcRd7PugpdD7/0jgBiHk8EjcoqHfTF9EXpysF+6RW8MQtY4pyBpw/cHaJ\n+DB3JugzQYDLY/FSIyDXpu0u3V7qjtdsxA==\n=7tq6\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:32 :DEBUG:[GNUPG:] NEED_PASSPHRASE_SYM 9 3 2
2019/05/15 11:45:32 :DEBUG:[GNUPG:] BEGIN_ENCRYPTION 2 9
2019/05/15 11:45:32 :DEBUG:[GNUPG:] END_ENCRYPTION
2019/05/15 11:45:32 :DEBUG:encrypt result: b'-----BEGIN PGP MESSAGE-----\n\njA0ECQMCcRd7PugpdD7/0jgBiHk8EjcoqHfTF9EXpysF+6RW8MQtY4pyBpw/cHaJ\n+DB3JugzQYDLY/FSIyDXpu0u3V7qjtdsxA==\n=7tq6\n-----END PGP MESSAGE-----\n'
2019/05/15 11:45:32 :INFO:Password is being inserted
2019/05/15 11:45:32 :INFO:Inserted data into passwords table
2019/05/15 11:45:32 :INFO:Getting everything from passTable
2019/05/15 11:45:32 :DEBUG:13494: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:32 :DEBUG:Wrote passphrase
2019/05/15 11:45:32 :DEBUG:data copier: <Thread(Thread-45, initial daemon)>, <_io.BytesIO object at 0x7f2926590258>, <_io.BufferedWriter name=27>
2019/05/15 11:45:32 :DEBUG:stderr reader: <Thread(Thread-46, initial daemon)>
2019/05/15 11:45:32 :DEBUG:closed output, 1 bytes sent
2019/05/15 11:45:32 :DEBUG:stdout reader: <Thread(Thread-47, initial daemon)>
2019/05/15 11:45:32 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:32 :DEBUG:gpg: [don't know]: invalid packet (ctb=31)
2019/05/15 11:45:32 :DEBUG:[GNUPG:] NODATA 3
2019/05/15 11:45:32 :DEBUG:decrypt result: b''
2019/05/15 11:45:32 :DEBUG:13498: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:32 :DEBUG:Wrote passphrase
2019/05/15 11:45:32 :DEBUG:data copier: <Thread(Thread-48, initial daemon)>, <_io.BytesIO object at 0x7f2926590258>, <_io.BufferedWriter name=27>
2019/05/15 11:45:32 :DEBUG:stderr reader: <Thread(Thread-49, initial daemon)>
2019/05/15 11:45:32 :DEBUG:closed output, 163 bytes sent
2019/05/15 11:45:32 :DEBUG:stdout reader: <Thread(Thread-50, initial daemon)>
2019/05/15 11:45:33 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:33 :DEBUG:gpg: AES256 encrypted data
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NEED_PASSPHRASE_SYM 9 3 2
2019/05/15 11:45:33 :DEBUG:gpg: encrypted with 1 passphrase
2019/05/15 11:45:33 :DEBUG:chunk: b'test'
2019/05/15 11:45:33 :DEBUG:[GNUPG:] BEGIN_DECRYPTION
2019/05/15 11:45:33 :DEBUG:[GNUPG:] DECRYPTION_COMPLIANCE_MODE 23
2019/05/15 11:45:33 :DEBUG:message ignored: DECRYPTION_COMPLIANCE_MODE, 23
2019/05/15 11:45:33 :DEBUG:[GNUPG:] DECRYPTION_INFO 2 9
2019/05/15 11:45:33 :DEBUG:[GNUPG:] PLAINTEXT 62 1557913404
2019/05/15 11:45:33 :DEBUG:[GNUPG:] PLAINTEXT_LENGTH 4
2019/05/15 11:45:33 :DEBUG:[GNUPG:] DECRYPTION_OKAY
2019/05/15 11:45:33 :DEBUG:[GNUPG:] GOODMDC
2019/05/15 11:45:33 :DEBUG:[GNUPG:] END_DECRYPTION
2019/05/15 11:45:33 :DEBUG:decrypt result: b'test'
2019/05/15 11:45:33 :DEBUG:13502: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:33 :DEBUG:Wrote passphrase
2019/05/15 11:45:33 :DEBUG:data copier: <Thread(Thread-51, initial daemon)>, <_io.BytesIO object at 0x7f2926590258>, <_io.BufferedWriter name=27>
2019/05/15 11:45:33 :DEBUG:stderr reader: <Thread(Thread-52, initial daemon)>
2019/05/15 11:45:33 :DEBUG:closed output, 26 bytes sent
2019/05/15 11:45:33 :DEBUG:stdout reader: <Thread(Thread-53, initial daemon)>
2019/05/15 11:45:33 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:33 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:33 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:33 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:33 :DEBUG:decrypt result: b''
2019/05/15 11:45:33 :DEBUG:13506: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:33 :DEBUG:Wrote passphrase
2019/05/15 11:45:33 :DEBUG:data copier: <Thread(Thread-54, initial daemon)>, <_io.BytesIO object at 0x7f2926590258>, <_io.BufferedWriter name=27>
2019/05/15 11:45:33 :DEBUG:stderr reader: <Thread(Thread-55, initial daemon)>
2019/05/15 11:45:33 :DEBUG:closed output, 5 bytes sent
2019/05/15 11:45:33 :DEBUG:stdout reader: <Thread(Thread-56, initial daemon)>
2019/05/15 11:45:33 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:33 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:33 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:33 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:33 :DEBUG:decrypt result: b''
2019/05/15 11:45:33 :DEBUG:13510: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:33 :DEBUG:Wrote passphrase
2019/05/15 11:45:33 :DEBUG:data copier: <Thread(Thread-57, initial daemon)>, <_io.BytesIO object at 0x7f2926590258>, <_io.BufferedWriter name=27>
2019/05/15 11:45:33 :DEBUG:stderr reader: <Thread(Thread-58, initial daemon)>
2019/05/15 11:45:33 :DEBUG:closed output, 13 bytes sent
2019/05/15 11:45:33 :DEBUG:stdout reader: <Thread(Thread-59, initial daemon)>
2019/05/15 11:45:33 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:33 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:33 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:33 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:33 :DEBUG:decrypt result: b''
2019/05/15 11:45:33 :DEBUG:13514: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:33 :DEBUG:Wrote passphrase
2019/05/15 11:45:33 :DEBUG:data copier: <Thread(Thread-60, initial daemon)>, <_io.BytesIO object at 0x7f2926590258>, <_io.BufferedWriter name=27>
2019/05/15 11:45:33 :DEBUG:stderr reader: <Thread(Thread-61, initial daemon)>
2019/05/15 11:45:33 :DEBUG:closed output, 4 bytes sent
2019/05/15 11:45:33 :DEBUG:stdout reader: <Thread(Thread-62, initial daemon)>
2019/05/15 11:45:33 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:33 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:33 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:33 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:33 :DEBUG:decrypt result: b''
2019/05/15 11:45:33 :DEBUG:13518: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:33 :DEBUG:Wrote passphrase
2019/05/15 11:45:33 :DEBUG:data copier: <Thread(Thread-63, initial daemon)>, <_io.BytesIO object at 0x7f2926590258>, <_io.BufferedWriter name=27>
2019/05/15 11:45:33 :DEBUG:stderr reader: <Thread(Thread-64, initial daemon)>
2019/05/15 11:45:33 :DEBUG:closed output, 5 bytes sent
2019/05/15 11:45:33 :DEBUG:stdout reader: <Thread(Thread-65, initial daemon)>
2019/05/15 11:45:33 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:33 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:33 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:33 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:33 :DEBUG:decrypt result: b''
2019/05/15 11:45:33 :DEBUG:13522: gpg --pinentry-mode loopback --status-fd 2 --no-tty --no-verbose --fixed-list-mode --batch --with-colons --homedir gpg --passphrase-fd 0 --decrypt
2019/05/15 11:45:33 :DEBUG:Wrote passphrase
2019/05/15 11:45:33 :DEBUG:data copier: <Thread(Thread-66, initial daemon)>, <_io.BytesIO object at 0x7f2926590258>, <_io.BufferedWriter name=27>
2019/05/15 11:45:33 :DEBUG:stderr reader: <Thread(Thread-67, initial daemon)>
2019/05/15 11:45:33 :DEBUG:closed output, 4 bytes sent
2019/05/15 11:45:33 :DEBUG:gpg: WARNING: unsafe permissions on homedir '/home/jfk422/Projects/Code/Arcanum/arcanum/gpg'
2019/05/15 11:45:33 :DEBUG:stdout reader: <Thread(Thread-68, initial daemon)>
2019/05/15 11:45:33 :DEBUG:gpg: no valid OpenPGP data found.
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 1
2019/05/15 11:45:33 :DEBUG:[GNUPG:] NODATA 2
2019/05/15 11:45:33 :DEBUG:[GNUPG:] FAILURE decrypt 4294967295
2019/05/15 11:45:33 :DEBUG:gpg: decrypt_message failed: Unknown system error
2019/05/15 11:45:33 :DEBUG:decrypt result: b''
